client/src/components/pages/MembersPage/components/ControlBar.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components'
+
+export function ControlBar({ toggleForm }) {
```

Comment:
Better supply a disabled prop and switch the style of your styled components accordingly instead of exporting two different Components that only differ in style.